napari/layers/image/image.py

```diff
@@ -450,7 +450,7 @@ def _get_state(self):
             {
                 'rgb': self.rgb,
                 'multiscale': self.multiscale,
-                'colormap': self.colormap[0],
+                'colormap': self.colormap.name,
```

Comment:
What do you think about storing a dictionary with the kwargs required to make the `Colormap` object? That way even custom colormaps could be reloaded from the `layer._get_state()` output.

Reply:
as briefly discussed before... I was kinda hoping this would actually just be the colormap object.  While that may not be immediately serializable (which I know was the original intended use for `_get_state`), truth is we've never actually used it that way, and now it forms a main part of the layer_data_tuple that plugins might get.  If we give them the object, they immediately have something useful.  If we just give them the name (or stuff to construct it), then they have to either depend on some of our convenience functions, or recreate them.  Serialization can be solved if/when we've ever truly need it, but passing the actual object would be immediately useful in (e.g.) napari-svg right?